doc/source/support/frequently-asked-questions.rst

```diff
@@ -134,6 +134,72 @@ Achieving the same result in Markdown often requires raw HTML, which is less rea
 
 Switching to ``README.rst`` at Level 5 helps users appreciate the formatting power of ``.rst`` and serves as a stepping stone toward writing full documentation in ``.rst`` as part of the ``scikit-package`` documentation standard.
 
+.. _faq-set-default-prompt-value:
+
+How can I change the default values that appear in the prompt?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To change the default values, create ``~/.skpkgrc``:
+
+.. code-block:: text
+
+  {
+      "default_context":
+          {
+	       "entry-name": "new-default-value",
+	       "entry-name": "new-default-value",
+	       # ...
+	  }
+  }
+
+
+Example ``~/.skpkgrc`` used by ``system`` templates for Level 4 usage.
+
+.. code-block:: text
+
+  {
+      "default_context":
+          {
+	    "project_name": "my-package",
+	    "github_username_or_orgname": "billingegroup",
+            "contributors": "Sangjoon Lee, Simon Billinge",
+	  }
+  }
+
+Example ``~/.skpkgrc`` used by ``public`` templates for Level 5 usage.
+
+.. code-block:: text
+
+    {
+	"default_context":
+	    {
+		"maintainer_name": "Simon Billinge",
+		"maintainer_email": "sb2896@columbia.edu",
+		"maintainer_github_username": "sbillinge",
+		"contributors": "Sangjoon Lee, Simon Billinge, Billinge Group members",
+		"license_holders": "The Trustees of Columbia University in the City of New York",
+		"project_name": "diffpy.my-project",
+		"github_username_or_orgname": "diffpy",
+		"project_short_description": "Python package for doing science.",
+		"project_keywords": "diffraction, PDF, X-ray, neutron",
+		"minimum_supported_python_version": "3.11",
+		"maximum_supported_python_version": "3.13",
+		"project_needs_c_code_compiled": ["No", "Yes"],
```

Comment:
Let's not use the default values.

Let's copy and paste what I shared in the PR (real example). We also don't have to give all the value